primefaces-integration-tests/src/test/java/org/primefaces/integrationtests/datatable/DataTable026Test.java

```diff
@@ -302,6 +303,58 @@ public void testFilterIn(Page page) {
         assertConfiguration(dataTable.getWidgetConfiguration());
     }
 
+    @Test
+    @Order(13)
+    @DisplayName("DataTable: filter: between LocalDateTime")
+    public void testFilterBetweenLocalDateTime(Page page) {
+        // Arrange
+        DataTable dataTable = page.dataTable;
+
+        // Act
+        LocalDate start = LocalDate.of(2021, 1, 1);
+        LocalDate end = LocalDate.of(2021, 1, 10);
+        page.lastLoginDateTimeFilter.getInput().sendKeys("" + start + " - " + end); // 2021-01-01 - 2021-01-10
+        PrimeSelenium.guardAjax(page.lastLoginDateTimeFilter.getInput()).sendKeys(Keys.TAB);
+
+        // Assert
+        List<Employee> employeesFiltered = employees.stream()
+                .filter(e -> {
+                    LocalDate date = e.getLastLoginDateTime().toLocalDate();
+                    return date.equals(start) || date.equals(end) || (date.isAfter(start) && date.isBefore(end));
+                })
+                .collect(Collectors.toList());
+        assertEmployeeRows(dataTable, employeesFiltered);
+
+        // Setting range via Selenium (sendKeys) causes JS-errors. So we don´t check for them.
+//        assertConfiguration(dataTable.getWidgetConfiguration());
```

Comment:
Should we investigate this?

Reply:
It was already like that for the between check of `LocalDate`s. But I can have a look what I get logged in my JS console.

Reply:
@melloware I've create a new ticket for this: #8362

Reply:
Excellent I will mark this convo resolved